packages/app/src/components/market/MarketListings.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { ReactElement, useState } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import { BlurView } from '@react-native-community/blur';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { TouchableOpacity, FlatList } from 'react-native-gesture-handler';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import { Button, Card, colors, LoadingSpinner, Title, XdaiBanner } from '../UI';
+import Animated, {
+    useSharedValue,
+    useAnimatedStyle,
+    withTiming
+} from 'react-native-reanimated';
+
+type Props = {
+    setAnimation: (b: boolean) => void;
+};
+
+export const MarketListings = ({ setAnimation }: Props): ReactElement => {
+    const insets = useSafeAreaInsets();
+    const opacity = useSharedValue(0);
+
+    // test data and placeholder loading functionality
+    const test = Array.from(Array(6).keys());
+    const [data, setData] = useState(false);
```

Comment:
Whats going on with this `data` state variable?

Reply:
Having the Flatlist load immediately completely ruins the fade in animation. The data variable is just a placeholder until the actual list data is hooked up, allowing the flatlist to render after the initial fade in.